demos/demos_databases_apis/microsoft/kusto/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c80d3a57-7c78-4bc3-bb4d-6b8cbd71f9fc",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Manage graphs with Kusto Persistent Graph & Graphistry\n",
+    "\n",
+    "## Microsoft Azure Data Explorer Update\n",
+    "\n",
+    "In a significant step forward for graph analytics, **Microsoft has introduced Persistent Graph Semantics in Azure Data Explorer (ADX)**.  \n",
+    "This feature, announced in [this official Azure update](https://azure.microsoft.com/en-us/updates?id=495985), allows users to define **graph projections as persistent database objects**. \n",
+    "\n",
+    "### Key Benefits:\n",
+    "- Define graph relationships using persisted `graph semantics` that live alongside your ADX tables.\n",
+    "- Simplifies repeatable graph analysis workflows.\n",
+    "- Enables performance optimizations by reusing defined graph topologies.\n",
+    "\n",
+    "This new capability is especially powerful for users working with large-scale telemetry, security, and relational datasets natively in Kusto Query Language (KQL).\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Graphistry Adopts ADX Persistent Graph Semantics\n",
+    "\n",
+    "The graph visualization and investigation platform **Graphistry** has embraced this enhancement in its latest release.\n",
+    "\n",
+    "### What’s New in Graphistry:\n",
+    "- **Native KQL Support**: You can now use KQL directly within Graphistry to run ADX queries and return dataframes.\n",
+    "- **Persistent Graph Queries**: Pass the name of your persistent graph and perform investigations in Graphistry's GPU-accelerated visual interface..\n",
+    "\n",
+    "\n",
+    "### Why It Matters:\n",
+    "With persistent graph semantics and Graphistry’s GPU-accelerated visual interface, analysts can now: Query, explore, and visualize ADX graph data with minimal friction.\n",
+    "\n",
+    "---\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a761a5b8-5a67-4d18-a72f-f04387aad089",
+   "metadata": {},
+   "source": [
+    "# Setup\n",
+    "\n",
+    "Install pygraphistry and the [Kusto python client](link), and get a free [Graphistry Hub GPU API key](https://hub.graphistry.com) or run your own [server](https://www.graphistry.com/get-started)\n",
+    "\n",
+    "\n",
+    "Install Graphistry and start your Kusto graph journey.\n",
+    "```bash\n",
+    "# Just Graphistry; bring your own Kusto install\n",
+    "pip install graphistry\n",
+    "\n",
+    "# Bundled Kusto install\n",
+    "pip install graphistry[kusto]\n",
+    "\n",
+    "```\n",
+    "---\n",
+    "Python:\n",
+    "```python\n",
+    "import graphistry\n",
+    "\n",
+    "graphistry.register()\n",
+    "graphistry.configure_kusto(cluster=KUSTO_CONF['cluster'], database=KUSTO_CONF['database'])\n",
+    "\n",
+    "df_5 = graphistry.kql(\"YourTable | take 5\")\n",
+    "\n",
```

Comment:
This was confusing as duplicates actual later, and wrong empty register call, can go away bc install section is shown next as actual tutorial?

If I wanted to say more in the install, it'd be to link to the register docs/tutorial

The kusto install stuff is useful, assuming not also duplicates...